UNRELEASED.md

```diff
@@ -0,0 +1,5 @@
+**Changed**
+
+- bpk-component-ticket
+  - Added new box-shadow
```

Comment:
what's this meaning?

Reply:
Thanks, It means box-shadow for stub part to support a dividing line.

Reply:
Updated it, thanks.